packages/@textlint/markdown-to-ast/test/fixtures/footnote-duplicate.text/output.json

```diff
@@ -6,82 +6,22 @@
             "children": [
                 {
                     "type": "Str",
-                    "value": "The NATO phonetic alphabet",
+                    "value": "The NATO phonetic alphabet[^wiki].",
```

Comment:
- [ ] We need to care footnote
  - Some Markdown extensions use it, and it will affect sentence length.

```
The NATO phonetic alphabet[^wiki].

[^wiki]: Read more about it on wikipedia: <http://en.wikipedia.org/wiki/NATO_phonetic_alphabet>.
[^wiki]: Not to be confused with: <http://en.wikipedia.org/wiki/International_Phonetic_Alphabet>.
```

Reply:
https://github.com/remarkjs/remark-footnotes does it?